dev/examples/controls/bar.js

```diff
@@ -71,6 +71,32 @@ const regions2 = [
         end: 30
     }
 ];
+const regions3 = [
```

Comment:
nit: what is the difference between `regions`, `regions2` and `regions3`.  can we use a better variable name here?

Reply:
@mraplumb  Those are different set of regions that we use in different scenarios. Removed region2 and region3 and added everything in single array(regions). you can see changes in [here](https://github.com/cerner/carbon-graphs/pull/300/commits/595865c1ef97c1cd837971b727d58b1e7d89c5c0#diff-5768493ee06baddc838b70d473847bb6R22).